File: isPrime.js
function total(arr){
    let sum = 0;
    for (let i = 0; i < arr.length; i++){
      if (checkPrime(i)){
        sum += arr[i];
      }
    }
     return sum;
   };
   function checkPrime(num, div = 2) {
       if (num <= 1) {
           return false;
       }
       if (div > Math.sqrt(num)) {
           return true;
       }
       if (num % div === 0) {
           return false;
       } 
       else {
           return checkPrime(num, div + 1);
       }
   }

File: is_prime.py
import math
def total(arr):
    sum = 0
    for i in range(len(arr)):
        if is_prime(i):
            sum += arr[i]
    return sum
    
def is_prime(n, i=2):
    if n <= 2:
        return n == 2
    if n % i == 0:
        return False
    if i * i > n:
        return True
    return is_prime(n, i + 1)